charts/fluent-bit/values.yaml

```diff
@@ -289,4 +289,9 @@ initContainers: []
 #     image: busybox
 #     command: ['do', 'something']
 
+initContainersTpl: []
```

Comment:
Can we instead just template initContainers instead of have both initContainers and initContainersTpl ? Or would the templating cause some breakage if we did it on the existing initContainers?

Reply:
It could break stranges config: _old_ initContainers with `{{ golang template }}`.

Here is a sample

```yaml
initContainers:
- name: sample-160
  image: gomplate
  commands:
  - bash
  - -c
  - |-
    cat <<EOF | gomplate
    Hello {{.Env.World | default "world"}}
    EOF
```

(I am writting this comment with my phone, the previous code may have typo)

Reply:
Thanks, that is a strange config indeed. I think I am willing to reuse existin `initContainer`

If someone did have the above config, could they adapt a fix to the breaking change?

Also nice coding on your phone!

Reply:
They can adapt it.
IMO the secure way is to escape the `{{` and `}}` using `{{"{{"}}` and ``{{"}}"}}`. But the code would become odd.
I suggest, depending of their script/tool, to change the delimiters (with `{%` and `%}`?)

I will update the PR tomorrow.

Thank you for the review 👍